c++/src/kj/miniposix.h

```diff
@@ -44,6 +44,8 @@
 #include <sys/types.h>
 #endif
 
+#include <sys/uio.h>
```

Comment:
Fixed.  No worries about the slow-turnaround.  Haven't been the quickest on my end either.